sites/all/themes/orwell/sass/bootstrap/_mixins.scss

```diff
@@ -183,29 +182,32 @@
   line-height: 1em;
   padding: 10px 22px;
   margin-top: 1rem;
-  background-color: $color__green;
+  background-color: $color__button-green--dark;
   color: $color__white;
```

Comment:
<img width="401" alt="Screenshot 2020-05-07 at 09 04 51" src="https://user-images.githubusercontent.com/15377965/81267687-2705f100-9047-11ea-81e9-9ca180d96bb3.png">
Kønne ka' man ikke kalde dem...

Reply:
Is the hover state still visible?

Reply:
Yess

Reply:
![image](https://user-images.githubusercontent.com/15377965/85692540-c786a200-b6d5-11ea-899d-5bc0ca0fbb09.png)

Det er lån som er hoveret. igen, kønt er det ikke!